tox.ini

```diff
@@ -315,6 +319,7 @@ commands =
     gevent{11,12,13}: nosetests {posargs} tests/contrib/gevent
     gevent{10}: nosetests {posargs} tests/contrib/gevent
     httplib: nosetests {posargs} tests/contrib/httplib
+    influx{30,40,41,50,51,52}: nosetests {posargs} tests/contrib/influx
```

Comment:
We very recently changed the way we define commands vs dependencies, to avoid the tests for the _command_ `influx50` to be run when another command define the `influx50`.
So please change this line to `influx_contrib: pytest {posargs} tests/contrib/influx ` (similar to [sqlite3](https://github.com/DataDog/dd-trace-py/blob/master/tox.ini#L301)).

Please note also that we are trying to, progressively, replace `nose` with `pytest`.

Reply:
Oh wow, all of that changed _11 hours ago_. Let me follow suit...

Reply:
Changed in commit 1faa2ab7827836c122d71089f3d517a321ffd410